src/mutation_data.cpp

```diff
@@ -132,7 +132,7 @@ void mutation_branch::load( JsonObject &jsobj )
         int neutral = jo.get_int("neutral", 0);
         int good = jo.get_int("good", 0);
         tripoint protect = tripoint(ignored, neutral, good);
-        new_mut.protection[part_id] = mutation_wet(body_parts[part_id], protect);
+        new_mut.protection[part_id] = mutation_wet(get_body_part_token( part_id ), protect);
```

Comment:
> Well, does the mutation-wetness effect actually work?

I didn't skip through the debugmsgs. But looking at the code, it should neither work in master nor here.